src/System.Private.CoreLib/shared/System/Globalization/GregorianCalendarHelper.cs

```diff
@@ -182,7 +182,7 @@ private int GetYearOffset(int year, int era, bool throwOnError)
                     {
                         throw new ArgumentOutOfRangeException(
                                     nameof(year),
-                                    string.Format(
+                                    SR.Format(
                                         CultureInfo.CurrentCulture,
```

Comment:
Do we really need to pass in `CultureInfo.CurrentCulture` explicitly in all these places? I would expect that `string.Format` is going to use current culture by default.

Reply:
Agreed.  I've gone through and removed those.  I also realized I neglected to look at src/, only shared/, so I've cleaned up that, too.